### DISCUSSION TITLE
Possible to use Google login with custom domain on Supabase?

### DISCUSSION THREAD

**Discussion by lancejpollard** [supabase/supabase#30305] [Questions]
Cross-posting [Supabase custom domains says foo.bar.com, but google says "only top level domains"?](https://stackoverflow.com/questions/79160573/supabase-custom-domains-says-foo-bar-com-but-google-says-only-top-level-domain)

Is it possible to use Google login with custom domain on Supabase?

**Comment by bytaesu** [comment]
When implementing social login with Supabase, the provider does not redirect directly to your domain. Instead, it redirects to the callback URL of your Supabase instance, which means you cannot customize the callback URL freely.

However, you can customize your Supabase instance URL by using vanity subdomains (custom.supabase.co) or a custom domain (custom.your-domain.com). This feature is available for projects on the Pro plan or higher.

Summary: It is not possible to fully customize the callback URL. To customize your Supabase instance URL, use the Pro plan or higher.

https://supabase.com/docs/guides/platform/custom-domains
